### ISSUE TITLE
"Error: Authorization page could not be loaded"

### ISSUE THREAD

**Issue by EfficiencyVI** [Wassup789/YouTube-Notifications#16]
Hi, I have 2 user profiles for Youtube under the same google account. I want to import the subscription for the secondary account that gives me the error above. The first user profile seems to work but it shows me my (one) subscriptions multiple times which is weird.

Unfortunately I cannot provide any additional informations. The console and the network informations do not show any errors.

**Comment by Wassup789** [comment]
I think I'd need more information about your secondary account, is your account a Google+ account or a YouTube account?

**Comment by EfficiencyVI** [comment]
It is a page. This is the Authentification page.
![data](https://cloud.githubusercontent.com/assets/8409366/19410731/b283e4bc-92f3-11e6-8000-340f59956660.png)
![data1](https://cloud.githubusercontent.com/assets/8409366/19410732/b289fcf8-92f3-11e6-9f4f-5523996ef26b.png)

The message appears after I click "allow".

**Comment by Wassup789** [comment]
Alright I think I might know what your issue is, the account you are attempting to login with is not a valid Youtube account. As you can see in the following screenshots, I am attempting to login with the account with the red outline but it does not exist in the second screenshot which tells me which YouTube accounts I have on this email.

![chrome_2016-10-15_12-19-04](https://cloud.githubusercontent.com/assets/1007707/19412783/f25bb9ba-92d1-11e6-8a52-abda10d0dedf.png)
![chrome_2016-10-15_12-19-56](https://cloud.githubusercontent.com/assets/1007707/19412782/f25a7aaa-92d1-11e6-94c7-55e56e169f94.png)

**Comment by EfficiencyVI** [comment]
Well, I can post comments, subscribe to people and even some people follow me so for Youtube it is a Youtube account. I created that account when Youtube integrated Google Plus into Youtube and I never had issues with it. I just don't want to maintain two lists for this. I just want to get notified when somebody uploads a video.

**Comment by Wassup789** [comment]
Okay, I'll need you to:
1. Goto `chrome://extensions`
2. Check `Developer Mode` on the top right
3. Find `YouTube Notifications`
4. Find `Inspect views:` and click on `background.html`
5. Goto the `Network` tab and make sure `XHR` is selected
6. Attempt to login again with the inspector window open
7. Once the error pops up again, scroll down to the bottom and click on the row that is RED.
8. Click on the `Response` tab and send the entire output

The layout of the inspector window should look similar to this:
![](https://cloud.githubusercontent.com/assets/1007707/19413227/4c617a76-92dc-11e6-9fc6-19f1948b0f0b.png)

**Comment by EfficiencyVI** [comment]
Nope, I have nothing like this when I follow the steps. But that is actually what I did before because I'm used to write my own addons and it does not show me any request/errors.

![data2](https://cloud.githubusercontent.com/assets/8409366/19414170/a95b591e-9344-11e6-8c77-634b9367323a.png)

**Comment by Wassup789** [comment]
Sorry for the late reply, it should definitely show the request there because that's the only way you would receive the error otherwise there would be no popup stating there's an error. Did you leave the background page open whilst clicking import? Also check the settings popup too just in case.

**Comment by tanxestudio** [comment]
FIXED: If youtube is blocked at your network, you will get error Authorization page could not be loaded.
